Makefile

```diff
@@ -76,7 +76,7 @@ serve:
 	node server.js
 
 test:
-	./node_modules/.bin/jasmine-node test/katex-spec.js
+	./node_modules/.bin/jasmine-node test/*-spec.js
```

Comment:
Chai has an [`expect` assertion style](http://chaijs.com/guide/styles/#expect), which looks more usable than unexpected and matches other assert libraries better. I think switching to mocha+chai sounds good, I'm not so sure about using expect vs assert, but I'd be willing to try it!